onnxruntime/python/session.py

```diff
@@ -255,7 +256,7 @@ def bind_cpu_input(self, name, arr_on_cpu):
     def bind_input(self, name, device_type, device_id, element_type, shape, buffer_ptr):
```

Comment:
> def bind_input [](start = 3, length = 15)

Is there anyway we deprecate those old APIs and create only those based on OrtValue? Essenstially, it is a duplicate code.

Reply:
Well - these older APIs are still useful in that the users could still use a custom allocator (say using Torch) and feed in a data pointer. While this isn't necessarily prescribed, it doesn't make sense to inhibit users from doing so should they wish. Moreover, the new APIs are meant to be complementary and not necessarily mean to replace existing method.